post.hbs

```diff
@@ -131,6 +151,11 @@ into the {body} of the default.hbs template --}}
 
 {{!-- The #contentFor helper here will send everything inside it up to the matching #block helper found in default.hbs --}}
 {{#contentFor "scripts"}}
+
+{{#if @labs.members}}
+    <script src="{{asset "js/members-theme-dropin.min.js"}}"></script>
```

Comment:
Should this be `built/members-theme-dropin.min.js`?

Reply:
I also think this should be in `default.hbs`

Reply:
Yeah it can go there, it already happens implicitly right now as everything in this block is moved to `defaults.hbs`.

Reply:
Yeah, but it'll only happen on a post right, we want the log in/ out buttons on every page afaik :)

Reply:
Ah yes, lets move it 👍